### PR TITLE
bump kernel to 6.11.6-2 + add landlock to `CONFIG_LSM` to enable it

### PR DESCRIPTION
**Pull request by mkurz**
Upgrading to latest kernel [tagged yesterday](https://github.com/AsahiLinux/linux/tags) - tested and works for me.

Regarding landlock, see https://docs.kernel.org/userspace-api/landlock.html#kernel-support:

> Landlock must also be enabled at boot time as the other security modules. The list of security modules enabled by default is set with `CONFIG_LSM`. The kernel configuration should then contains `CONFIG_LSM=landlock,[...]` with `[...]` as the list of other potentially useful security modules for the running system (see the `CONFIG_LSM` help).

### CODE REVIEW

File: linux-asahi/config [hunk 1]
```diff
@@ -7428,7 +7431,7 @@ CONFIG_SECURITY_LANDLOCK=y
 # CONFIG_IMA_SECURE_AND_OR_TRUSTED_BOOT is not set
 # CONFIG_DEFAULT_SECURITY_APPARMOR is not set
 CONFIG_DEFAULT_SECURITY_DAC=y
-CONFIG_LSM="yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor"
+CONFIG_LSM="landlock,yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor"
```
Comment (mkurz) [inline linux-asahi/config:7434]: Here I enabled `landlock`

File: linux-asahi/PKGBUILD [hunk 2]
```diff
@@ -3,12 +3,12 @@
 
 buildarch=8
 
-_rcver=6.11.4
+_rcver=6.11.6
 #_rcrel=3
-_asahirel=1
-pkgrel=2
+_asahirel=2
+pkgrel=1
```
Comment (mkurz) [inline linux-asahi/PKGBUILD:9]: @joske This `pkgrel` should have been `1` already in you yesterday's changes...

**Review by joske — APPROVED** [review]

**Comment by joske** [comment]
@mkurz thanks. Yesterday I noticed that audio no longer works on my system. Do you have the same?

**Comment by mkurz** [comment]
@joske Yes I had the same. There is an issue already:
- https://github.com/AsahiLinux/asahi-audio/issues/48
You can make it work again by downgrading lsp plugins to 1.2.17:
```
sudo pacman -U /var/cache/pacman/pkg/lsp-plugins-lv2-1.2.17-1-aarch64.pkg.tar.xz /var/cache/pacman/pkg/liblsp-r3d-glx-lib-1.2.17-1-aarch64.pkg.tar.xz
```

```
$ pacman -Q | grep -iE 'lsp'
liblsp-r3d-glx-lib 1.2.17-1
lsp-plugins-lv2 1.2.17-1
```

BTW, I think I ran into another problem regarding mesa. Can you check: Is your system up to date? Like are you running latest kwin 6.2.2-1 (`pacman -Q | grep -iE 'kwin')?
1. If so can you run `xeyes` - are they eyes moving if you move over it with the mouse?
2. Also, if you have `gitk` installed, can you go into a local git repo and just run it (`gitk`) - does it work for you or you have many artifacts?

**Comment by mkurz** [comment]
@joske Another topic: If you are using Plasma 6.2, does `ksystemstats` keep crashing for you? If so, I can help you with that as well.
It's a known issue already:
- https://oftc.irclog.whitequark.org/asahi/2024-10-29#1730241059-1730242297;
- https://bugs.kde.org/show_bug.cgi?id=495524

I rebuild `ksystemstats` with following patch applied and the crashes are gone:
- https://invent.kde.org/plasma/ksystemstats/-/merge_requests/92

**Comment by mkurz** [comment]
For the mesa issues [I talked about here](https://github.com/joske/PKGBUILDs/pull/11#issuecomment-2454212588) we need to upgrade to asahi-mesa 20241104, see #12.

**Pull request merged**